S7.Net.UnitTest/S7NetTests.cs

```diff
@@ -407,19 +430,40 @@ public void T10_ReadMultipleBytes()
                     VarType = VarType.Word
                 },
                 new DataItem
+                {
+                    Count = 1,
+                    DataType = DataType.DataBlock,
+                    DB = 2,
+                    StartByteAdr = 8192,
+                    BitAdr = 7,
+                    VarType = VarType.Bit
+                },
+                new DataItem
                 {
                     Count = 1,
                     DataType = DataType.DataBlock,
                     DB = 2,
                     StartByteAdr = 16,
                     VarType = VarType.Word
-                }
+                },
+                new DataItem
+                {
+                    Count = 1,
+                    DataType = DataType.DataBlock,
+                    DB = 2,
+                    StartByteAdr = 16384,
+                    BitAdr = 6,
```

Comment:
This is undefined.

Reply:
Added BitAdr to S7.Net/Types/DataItem.cs, was lost in previous merge.